conferences/kubecon-eu-2025.yaml

```
@@ -30,6 +30,16 @@ parties:
     Dagger HackNight brings together developers and devops professionals of all levels to explore the possibilities of building and sharing CI/CD pipelines with Dagger. From hands-on labs to advanced projects, this event is designed to inspire and empower every attendee.
   notes: |
 
+- name: House of Kube
+  date: 2025-04-01
+  time: "7pm - 2:00am"
+  website: https://www.houseofkube.com/
+  location: TBA
```

```suggestion
  location: London
```

Just to make sure the map link is valid :)